posix/clone

```diff
@@ -4,11 +4,15 @@ if [[ ! -z "${DRONE_WORKSPACE}" ]]; then
 	cd ${DRONE_WORKSPACE}
 fi
 
+if [ -z "${HOME}" ]; then
+	HOME=/root
```

Comment:
suggest that we export the home directory so that we do not need to include this same code block in all the other scripts invoked from this script.

```diff
-HOME=/root
+export HOME=/root
```